sources/packages/backend/libs/utilities/src/date-utils.ts

```diff
@@ -53,13 +53,16 @@ export const isAfter = (
  * Difference in days between endDate and startDate (endDate-startDate).
  * @param endDate end date.
  * @param startDate start date.
+ * @param inclusiveDiff whether it should consider start and end date as part of the calculation.
```

Comment:
For my understanding, in which case might this parameter be set to false?

Reply:
As discussed in our meeting, all the dates will be inclusive, then I removed the param. Thanks for the question.